src/test/java/io/cdap/plugin/gcp/bigquery/sink/BigQuerySinkUtilsTest.java

```diff
@@ -522,4 +524,74 @@ public void testGetRelaxedTableFieldsMarksFieldsNullable() {
     Assert.assertEquals("c", result.get(2).getName());
     Assert.assertEquals(Field.Mode.REPEATED, result.get(2).getMode());
   }
+
+
+  @Test
+  public void testGenerateUpdateUpsertQueryUpdate() {
+    Operation operation = Operation.UPDATE;
+
+    TableId sourceTableId = TableId.of("akritigiri-gcp-project0", "test",
```

Comment:
nit: Is this an actual project or just some value you made up? Generally, we don't put references to actual resources in the code.

Reply:
An actual project